proto/panacea/datapool/v2/query.proto

```diff
@@ -12,6 +12,11 @@ service Query {
   rpc Pool(QueryPoolRequest) returns (QueryPoolResponse) {
     option (google.api.http).get = "/panacea/datapool/v2/pools/{pool_id}";
   }
+
+  // Contract returns a contract address registered to x/datapool module
+  rpc Contract(QueryContractRequest) returns (QueryContractResponse) {
+    option (google.api.http).get = "/panacea/datapool/v2/contract";
```

Comment:
It would be better to name this as `.../nft-contract`

Reply:
renamed in [abecf64](https://github.com/medibloc/panacea-core/pull/288/commits/abecf64bd84313eb2a8de05373eb08f76fe860c8)

Reply:
Ah. Sorry. Please use underscore instead of hyphen: `.../nft_contract`.

Reply:
Also, I would put the prefix `NFT` to all `Contract` symbols such as function names, rpc names and so on.

Reply:
I got it. It would be clearer. Let me add `NFT` prefix to all `Contract`

Reply:
done in [d9ff16a](https://github.com/medibloc/panacea-core/pull/288/commits/d9ff16a1f4e2195a4613f30d8cc5437ae32ef292)